scripts/xdist/get_worker_test_list.py

```diff
@@ -0,0 +1,51 @@
+"""
+This script strips the console log of a pytest-xdist Jenkins run into the test
+lists of each pytest worker.
+Assumes the following format:
+[test-suite] [worker] RESULT test
+"""
+import click
+import io
+import re
+import os
+import shutil
+
+
+@click.command()
+@click.option(
+    '--log-file',
+    help="File name of console log .txt file from a Jenkins build "
+    "that ran pytest-xdist. This can be acquired by running: "
+    "curl -o console.txt https://build.testeng.edx.org/job/JOBNAME/BUILDNUMBER/consoleText",
+    required=True
+)
+@click.option(
+    '--test-suite',
+    help="Test suite that the pytest worker ran. Example: lms-unit",
+    required=True
+)
+def main(log_file, test_suite):
+    worker_test_dict = {}
+    with io.open(log_file, 'r') as console_file:
+        for line in console_file:
+            regex_search = re.search(r'\[{}] \[gw(\d+)] PASSED|FAILED|SKIPPED'.format(test_suite), line)
```

Comment:
I think we need to account for "ERROR" as a result option also (if there's an exception during setup or teardown, etc.)

Reply:
good catch. done.